cmd/api/main.go

```diff
@@ -47,241 +47,106 @@ import (
 	"go.opentelemetry.io/otel/attribute"
```

Comment:
I'm not sure if looking at diffs here is fine, or it's better to see the real file.